sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Mean.java

```diff
@@ -101,7 +101,7 @@ private Mean() { } // Namespace only
 
   /////////////////////////////////////////////////////////////////////////////
 
-  private static class MeanFn<NumT extends Number>
+  static class MeanFn<NumT extends Number>
```

Comment:
necessary? or can you use [`Mean.<Double>of()`](https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Mean.java#L98) in the one place you do `new MeanFn<Double>()`?